### PR TITLE
Move status tests to sessions package

### PR DESCRIPTION
**Pull request by shs96c**
This makes it easier to find, as the tests are located
in a directory named after the section of the spec the
command is located in.

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None



View more information about this build on:

- [WPT Results Dashboard](http://pulls.web-platform-tests.org/build/17219)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/263137479)

File: tools/wpt/browser.py [hunk 1]
```diff
@@ -154,9 +154,9 @@ def install_prefs(self, dest=None):
         dest = os.path.join(dest, "profiles")
         if not os.path.exists(dest):
             os.makedirs(dest)
-        with open(os.path.join(dest, "prefs_general.js"), "wb") as f:
-            resp = get("https://hg.mozilla.org/mozilla-central/raw-file/tip/testing/profiles/prefs_general.js")
-            f.write(resp.content)
+        #with open(os.path.join(dest, "prefs_general.js"), "wb") as f:
+        #    resp = get("https://hg.mozilla.org/mozilla-central/raw-file/tip/testing/profiles/prefs_general.js")
+        #    f.write(resp.content)
```
Comment (andreastt) [inline tools/wpt/browser.py]: This probably shouldn’t be here.

**Review by andreastt — DISMISSED** [review]

File: tools/wpt/browser.py (hunk 1 above)
Comment (shs96c) [inline tools/wpt/browser.py]: Ha! Good point :)

**Comment by w3c-bots** [comment]
# Build ERRORED

Started: 2017-08-18 19:10:23
Finished: 2017-08-18 19:21:54


<h2>Failing Jobs</h2>
<ul>

<li>wpt_integration in py27</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/17228)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/263188009)

**Review by andreastt — APPROVED** [review]

**Comment by gsnedders** [comment]
@jgraham are we concerned by the FF failure here?

**Comment by andreastt** [comment]
I don’t understand why it would be _unstable_.

**Review by AutomatedTester — APPROVED** [review]

**Pull request merged**